target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -186,7 +186,8 @@ miniembplug)
 	set_wifi_led "$board:red:wlan"
 	set_usb_led "$board:green:mobile"
 	;;
-miwifi-mini)
+miwifi-mini|\
+miwifi-nano)
```

Comment:
I see that this was already handled (see 3 lines below). Why do you need this again here?